src/pages/dmt5/_interest_free_trading.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import Proptypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex, Show } from 'components/containers'
+import { CardStyle, Header, Text } from 'components/elements'
+import Checkmark from 'images/svg/checkmark.svg'
+import ZeroPercent from 'images/svg/zero_percent.svg'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+import { Localize } from 'components/localization'
+
+const ImageWrapper = styled.div`
+    max-width: 124px;
+    width: 100%;
+    object-fit: contain;
+    padding: 0;
+    margin: 4rem 0 0 0;
+`
+
+const InterestCardFlex = styled(Flex)`
+    width: 340px;
+    height: 392px;
+    border-radius: 8px;
+    margin-right: 12.6rem;
+    padding: 4rem;
+    ${CardStyle}
+`
+
+const StyledContainer = styled(Flex)`
+    background-color: var(--color-grey-25);
+    margin-bottom: 5.2rem;
+`
+
+const StyledHeader = styled(Header)`
+    margin-top: 15.4rem;
+    @media ${device.mobileL} {
+        font-size: ${(props) => props.mobile_font_size};
+        margin: ${(props) => props.mobile_margin};
+    }
+`
+
+const StyledLinkButton = styled(LinkButton)`
+    padding: 1rem 1.6rem;
+    font-size: 14px;
+    height: 4rem;
+    width: 24.1rem;
+
+    :active {
+        outline: none;
+        border: none;
+    }
+    :focus {
+        outline: 0;
+    }
+`
+
+const StyledText = styled(Text)`
+    display: flex;
+    align-items: center;
+`
+const StyledRightText = styled(Text)`
+    margin-bottom: 3.2rem;
+`
+
+const Separator = styled.div`
+    width: 100%;
+    height: 1px;
+    margin: 0.4rem 0 1.6rem 0;
+    background-color: var(--color-grey-21);
+`
+
+const TextWrapperFlex = styled(Flex)`
+    margin: 2.4rem 4rem 4rem 4rem;
+`
+
+const CheckedText = ({ children }) => (
+    <>
+        <StyledText size="16px" weight="bold">
+            <img src={Checkmark} alt="Check mark" width="24" height="24" /> {children}
+        </StyledText>
```

Comment:
This should be in a form of
```
<Flex ai="center">
    <Localize translate_text=... components=...> 
    <img ... />
</Flex>
```
So that the text and image will be wrapped with a container.
`Localize` in case children need to be translated